model_tests.rb

```diff
@@ -529,6 +529,15 @@ def test_generator_windturbine_osm
     result = sim_test('generator_windturbine.osm')
   end
 
+  def test_ghx_horizontal_trench_rb
+    result = sim_test('ghx_horizontal_trench.rb')
+  end
+
+  # TODO: To be added in the next official release after: 3.5.1
+  # def test_ghx_horizontal_trench_osm
+  #  result = sim_test('ghx_horizontal_trench.osm')
+  # end
```

Comment:
I think for the Kusuda one we actually want to make a test that runs on 3.5.1 (at least there) and commit that OSM. So we can test your VT and make sure results didn't change too